examples/solvers/bregman_tv_tomography.py

```diff
@@ -0,0 +1,95 @@
+"""Example of Bregman-TV in tomography.
+
+The example does seven iterations in a Bregman-TV scheme, applied to a
+tomography problem.
+"""
+
+import odl
+import numpy as np
+
+# Discrete reconstruction space
+reco_space = odl.uniform_discr(min_pt=[-20, -20], max_pt=[20, 20],
+                               shape=[128, 128], dtype='float32')
+
+# Make a parallel beam geometry with flat detector
+angle_partition = odl.uniform_partition(0, np.pi, 100)
+detector_partition = odl.uniform_partition(-30, 30, 558)
+geometry = odl.tomo.Parallel2dGeometry(angle_partition, detector_partition)
+
+# Create the ray transform
+ray_trafo = odl.tomo.RayTransform(reco_space, geometry)
+
+# Create phantom
+discr_phantom = odl.phantom.shepp_logan(reco_space, modified=True)
+
+# Create sinogram of forward projected phantom with noise
+noise_free_data = ray_trafo(discr_phantom)
+noise = odl.phantom.white_noise(ray_trafo.range)
+noise *= 1 / noise.norm() * noise_free_data.norm() * 0.10
+data = noise_free_data + noise
+
+# Do a FBP-reconstruction
+fbp_op = odl.tomo.fbp_op(ray_trafo, filter_type='Hann', frequency_scaling=0.4)
+fbp_reco = fbp_op(data)
+
+# Display phantom, FBP reconstruction, and sinograms
+discr_phantom.show(title='Phantom')
+fbp_reco.show(title='FBP reconstruction')
+data.show(title='Simulated data (Sinogram)')
+
+# Define components of the variational problem
+reg_param = 0.3
+gradient = odl.Gradient(reco_space)
+
+# l2-squared data matching
+l2_norm = 0.5 * odl.solvers.L2NormSquared(ray_trafo.range).translated(data)
+
+# Isotropic TV-regularization
+l1_norm = odl.solvers.GroupL1Norm(gradient.range)
+
+# Assembel functionals and operators for the optimization solver
+f = odl.solvers.ZeroFunctional(reco_space)  # Don't use f functional, set zero
+g = [l2_norm, reg_param*l1_norm]
+L = [ray_trafo, gradient]
+
+# Estimated operator norms, add 10 percent to ensure that we fulfill the
+# convergence criteria of the optomization algorithm
+ray_trafo_norm = odl.power_method_opnorm(ray_trafo)
+gradient_norm = odl.power_method_opnorm(gradient)
+
+# Set parameters for the optimization algorithm
+niter = 200  # Number of iterations in the inner loop
+tau = 0.01  # Step size for the primal variable
+sigma_ray_trafo = 45.0 / ray_trafo_norm**2  # Step size for the dual variable
+sigma_gradient = 45.0 / gradient_norm**2  # Step size for the dual variable
```

Comment:
Spaces around `**`

Reply:
Also the numbers are a bit "magical". Why not use the default stepsize helper? (It may not be implemented for forward-backward, though...)
Adding a parameter check would be nice for users.

Reply:
Since it is an iterative procedure inside an iterative procedure, I wanted to not have to  do to many inner iterations. Therefore I have tuned the numbers in order to be able to reduce `niter` to "only" 200. So the numbers are indeed a bit magical.

Taking something more standard as standard `tau = 0.5` and `sigma_i = 1 / op_norm**2` seems to demand many more inner iterations.

Reply:
Ok, fair enough. That would definitely be worth mentioning in a comment.